collection_requirements.rst

```diff
@@ -1,73 +1,76 @@
 
-**Note:** Inclusion of a new collection in the Ansible package is ultimately at the discretion of the Ansible community review committee. Every rejected candidate will get feedback. Differences of opinion should be taken to the community irc meeting for discussion and a final vote.
+.. note::
 
+  `Inclusion of a new collection <https://github.com/ansible-collections/ansible-inclusion>`_ in the Ansible package is ultimately at the discretion of the `Ansible Community Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_. Every rejected candidate will get feedback. Differences of opinion should be taken to a dedicated `Community Topic <https://github.com/ansible-community/community-topics/issues>`_ for discussion and a final vote.
 
 Feedback
 ========
 
-As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+As with any project it's very important that we get feedback from users, contributors, and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
 
 Please raise feedback by:
 
-* Discussing in the ``#ansible-community`` `libera.chat <https://libera.chat/>`_ IRC channel.
-* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_.
-* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repository.
+* Discussing in the ``#ansible-community`` Matrix/IRC `channel <https://docs.ansible.com/ansible/latest/community/communication.html#real-time-chat>`_.
+* Discussing in the `Community Working Group meeting <https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays>`_.
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ in this repository.
 
 Keeping informed
 ================
 
-Be sure you're subscribed to:
-
-* `Changes impacting Collections <https://github.com/ansible-collections/overview/issues/45>`_ to track changes that Collection maintainers should be aware of
-* The Bullhorn, a newsletter for the Ansible developer community, `back issues and how to add content <https://github.com/ansible/community/issues/546>`_
+Be sure you are subscribed to:
 
-Why is this needed
-===================
+* The `news-for-maintainers repository <https://github.com/ansible-collections/news-for-maintainers>`_ to track changes that Collection maintainers should be aware of.
+* The `Bullhorn <https://github.com/ansible/community/wiki/News#the-bullhorn>`_ Ansible contributor newsletter.
 
 Collection Infrastructure
 =========================
 
-* MUST have a publicly available issue tracker, that does not require a paid level of service to create an account or view issues.
-* Collections MUST have a Code of Conduct (CoC)
+Community collections
```

Comment:
Is this about `community.*` collections, or about collections included in the Ansible package? The below rules are about the latter, but this sounds like the former. How about:
```suggestion
Collections included in the Ansible community package
```

Reply:
Works for me:)